### PR TITLE
Update triage and prioritization processes

### PR DESCRIPTION
**Pull request by ppbaena**
### Description of the change

Update documentation and issue templates to update and standardize the issue creation, triage and prioritization processes.

### Benefits

Standard way to manage issues from maintainers. Information clear and updated for contributors.

### Possible drawbacks

Need for update the stale process.

### CODE REVIEW

File: site/content/docs/latest/reference/developer/issue-triage-process.md [hunk 1]
```diff
@@ -1,56 +1,85 @@
...
+| -------------------- | ---------------------------------------------------------------- |
+| `kind/bug`           | An issue that reports a defect in an existing feature            |
+| `kind/documentation` | An issue that reports an update related to project documentation |
+| `kind/proposal`      | An issue that reports a new feature proposal to be discussed     |
+| `kind/feature`       | An issue that reports a feature (approved) to be implemented     |
+| `kind/enhancement`   | An issue that reports an enhancement for an implemented feature  |
+
+### `component/*`:
+
+The area of the issue. Components are high level areas of Kubeapps architecture. They are used to group issues into smaller and related parts.
```
Comment (absoludity) [inline site/content/docs/latest/reference/developer/issue-triage-process.md:39]:
```suggestion
The relevant component(s) for the issue. Components are high level areas of the Kubeapps architecture. They are used to group issues together with other related issues.
```

File: site/content/docs/latest/reference/developer/issue-triage-process.md [hunk 2]
```diff
@@ -60,34 +89,50 @@ Besides this manual triage process, there are two automatic process lead by the
 | 'L'   | A story that requires investigation and possibly will take a person a full 2-week iteration |
 | 'XL'  | A story too big or with too many unknowns. Needs investigation and split into several ones  |
 
-### Priority
+| priority/ |       |       |       |
+| --------- | ----- | ----- | ----- |
+| ⛔️ P0    | 🔴 P1 | 🟠 P2 | 🔵 P3 |
```
Comment (absoludity) [inline site/content/docs/latest/reference/developer/issue-triage-process.md:94]: Might be worth a small description for each:

- P0 - drop everything and fix it
- P1 - ...

Fine either way.

**Review by absoludity — APPROVED** [review]
Thanks for documenting the process clearly @ppbaena ! Fine as is, a couple of small suggestions but feel free to ignore.

**Comment by castelblanque** [comment]
LGTM, thanks Pepe!

File: site/content/docs/latest/reference/developer/issue-triage-process.md [hunk 3]
```diff
@@ -1,56 +1,85 @@
...
-| kind/ |           |            |            |
-| ----- | --------- | ---------- | ---------- |
-| 'bug' | 'feature' | 'question' | 'refactor' |
+- The backlog is completely open. The maintainer team, along with the entire Kubeapps community, files all feature enhancements, bugs, and potential future work into the open repository.
+- Issues are closed if solved, or outdated (meaning the issue does not apply anymore according to the evolution of Kubeapps or it was waiting for more information which was never received).
 
-### Size
+## 🏷 Labeling
+
+Kubeapps mantainers team operates with 4 groups of labels:
```
Comment (antgamdia) [inline site/content/docs/latest/reference/developer/issue-triage-process.md:23]:
```suggestion
Kubeapps maintainers team operates with 4 groups of labels:
```

File: site/content/docs/latest/reference/developer/issue-triage-process.md [hunk 4]
```diff
@@ -1,56 +1,85 @@
...
+| ---------------------------- | ------------------------------------------------------------------------------------------------ |
+| `component/api-server`       | An issue related to kubeapps api-server                                                          |
+| `component/apprepository`    | An issue related to kubeapps apprepository                                                       |
+| `component/asset-syncer`     | An issue related to kubeapps asset-syncer (to be deprecated)                                     |
+| `component/assetsvc`         | An issue related to kubeapps assetsvc (to be deprecated)                                         |
+| `component/authentication`   | An issue related to kubeapps authentication                                                      |
+| `component/ci`               | An issue related to kubeapps ci system                                                           |
+| `component/kubeops`          | An issue related to kubeops (to be deprecated)                                                   |
+| `component/packages`         | An issue related to kubeapps packging formats to be distributed (Helm chart and Carvel packages) |
```
Comment (antgamdia) [inline site/content/docs/latest/reference/developer/issue-triage-process.md:50]:
```suggestion
| `component/packages`         | An issue related to kubeapps packaging formats to be distributed (Helm chart and Carvel packages) |
```

File: site/content/docs/latest/reference/developer/issue-triage-process.md [hunk 5]
```diff
@@ -1,56 +1,85 @@
...
+| `awaiting-more-evidence` | Need more info requested to actually get it done.                                |
+| `cla-notrequired`        | Automatic label for CLA signature                                                |
+| `cla-rejected`           | Automatic label for CLA signature when rejected                                  |
+| `dependencies`           | Automatic label set to pull requests that update a dependency file               |
+| `github_actions`         | Label assigned to pull requests that update GitHub Actions code                  |
+| `go`                     | Automatic label set to pull requests that update Go code                         |
+| `good first issue`       | Good first issues to start contributing to Kubeapps.                             |
+| `help wanted`            | The maintainer team wants help on an issue or pull request.                      |
+| `javascript`             | Automatic label set to pull requests that update javascript code                 |
+| `next-iteration`         | Label to mark issues to be discussed in next planning session                    |
```
Comment (antgamdia) [inline site/content/docs/latest/reference/developer/issue-triage-process.md:74]:
```suggestion
| `next-iteration`         | Label to mark issues to be discussed in the next planning session                    |
```

**Review by antgamdia — APPROVED** [review]

**Pull request merged**